src/index.ts

```diff
@@ -228,7 +228,10 @@ async function appBoot() {
 
 		createMainWindow();
 
-		bumpAggregatedUniqueStat( 'local-environment-launch-uniques', process.platform, 'weekly' );
+		// Bump a stat on each app launch
+		bumpStat( 'studio-app-launch', process.platform );
+		// Bump stat for unique weekly app launch, approximates weekly active users
+		bumpAggregatedUniqueStat( 'studio-app-launch-weekly-uniques', process.platform, 'weekly' );
```

Comment:
I'm very open to naming suggestions here. I went with a rename as it felt like the old name would cause confusion with the fact we're now going to track every launch separately.

Note, as a follow up to this PR, we'll need to merge the `local-environment-launch-uniques` stats into the new name. There is a dedicated page with guidance for this in the FG.